### PR TITLE
ZK-4565: Upgrade jQuery version to 3.5.1

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zk/src/archive/web/js/zk/ext/jquery.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
 /*!
- * jQuery JavaScript Library v1.12.4
- * http://jquery.com/
+ * jQuery JavaScript Library v3.5.0
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/ext/jquery.js:2]: 3.5.1

File: zk/src/archive/web/js/zk/ext/jquery.js [hunk 2]
```diff
@@ -8218,313 +8039,92 @@ jQuery.fn.delay = function( time, type ) {
 	} );
 };
 
+/* James Chu, Potix, add this back for compatible 20200514 */
+// The number of elements contained in the matched element set
+jQuery.fn.size = function() {
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/ext/jquery.js:8044]: .size => .length

File: zkdoc/release-note [hunk 3]
```diff
@@ -10,6 +10,7 @@ ZK 9.1.0
   ZK-4562: Introduce OWASP Dependency-Check
   ZK-4569: MVVM support calling commands in other view model
   ZK-4175: replace XHR with the fetch() API to allow SSO redirect handling
+  ZK-4565: Upgrade jQuery version to 3.5.0
```
Comment (DevChu) [inline zkdoc/release-note:13]: 3.5.1 and upgrade note

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**